lib/app/routes/account.rb

```diff
@@ -8,14 +8,16 @@ class Account < Core
         erb :"account/show", locals: { profile: Profile.new(current_user) }
       end
 
-      put '/account/email' do
+      put '/account' do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.